build-tools/automation/azure-pipelines.yml

```diff
@@ -0,0 +1,215 @@
+# Xamarin.Android Pipeline
+
+trigger:
+  - master
+  - d16-?
```

Comment:
Are these like filesystem globs or something else?  Specifically, will `?` match only a single digit, i.e. "only" up to `d16-9`, or will it match multiple digits, e.g.  also match `d16-10` and above?

If this *is* like a filesystem glob, any reason we shouldn't match `d16-*`?

Reply:
I asked because I have named a branch `d16-2-backport-foo` before; https://github.com/xamarin/xamarin-android/pull/3085#discussion_r287075868

Reply:
The `?` represents a [single character wildcard](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#wildcards), it won't work with a branch named `d16-10` or later. I am happy to revert back to the initial approach of a multiple character wildcard `*`, as I don't see much downside in automatically building commits to branches starting with `d16-` as this is likely a somewhat rare (or at least short lived) branch naming schema. Additionally, any branches matching this schema coming from a fork will not be automatically built either.